src/util.js

```diff
@@ -1,3 +1,5 @@
 export function getPcs() {
-	return game.actors.filter(actor => actor.data.type === "character").map(actor =>{return {id: actor.id, name: actor.data.name, image: actor.data.img}})
+	let actors = game.actors.filter(actor => actor.data.type === "character").map(actor =>{return {id: actor.id, name: actor.data.name, image: actor.data.img}});
+	let actors_aos = game.actors.filter(actor => actor.data.type === "player").map(actor =>{return {id: actor.id, name: actor.data.name, image: actor.data.img}});	
+	return actors.concat(actors_aos);
```

Comment:
This can be simplified into a single line by doing
`.filter(actor => actor.data.type === "character" || actor.data.type === "player)`